os/drivers/kdbg/Make.defs

```diff
@@ -0,0 +1,29 @@
+##########################################################################
+#
+# Copyright 2019 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+##########################################################################
+# Include kdbg drivers
+
+ifeq ($(CONFIG_KERNEL_CMDS),y)
+
+CSRCS += kdbg_drv.c
```

Comment:
Why is name kdbg? Is there any plan or possibility to support other kernel debug?